prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory.component.ts

```diff
@@ -204,4 +209,13 @@ export class RegulatoryComponent extends BaseEnrolmentProfilePage implements OnI
     const remoteLocations = this.enrolmentFormStateService.remoteAccessLocationsForm.get('remoteAccessLocations') as FormArray;
     [remoteAccessSites, enrolleeRemoteUsers, remoteLocations].forEach(f => f.clear());
   }
+
+  private enableDeviceProviderValidator(): void {
```

Comment:
This method is called enable but serves as both enable and disable. Should probably change the name. As well as the formatting on the below code is off

Reply:
Renamed to `toggleDeviceProviderValidator`